src/web/services/log.service.ts

```diff
@@ -62,19 +62,72 @@ export class LogService {
   searchLogs(queryParams: {
     searchFrom: string,
     searchUntil: string,
-    severities: string,
+    severity?: string,
+    minSeverity?: string,
+    logEvent?: string,
     nextPageToken?: string,
+    apiEndpoint?: string,
+    traceId?: string,
+    googleId?: string,
+    regkey?: string,
+    email?: string,
+    sourceLocationFile?: string,
+    sourceLocationFunction?: string,
+    exceptionClass?: string,
   }): Observable<GeneralLogs> {
     const paramMap: Record<string, string> = {
       starttime: queryParams.searchFrom,
       endtime: queryParams.searchUntil,
-      severities: queryParams.severities,
     };
 
+    if (queryParams.severity) {
+      paramMap.severity = queryParams.severity;
+    }
+
+    if (queryParams.minSeverity) {
+      paramMap.minseverity = queryParams.minSeverity;
+    }
+
+    if (queryParams.logEvent) {
+      paramMap.logevent = queryParams.logEvent;
+    }
+
     if (queryParams.nextPageToken) {
       paramMap.nextpagetoken = queryParams.nextPageToken;
     }
 
+    if (queryParams.apiEndpoint) {
+      paramMap.apiendpoint = queryParams.apiEndpoint;
```

Comment:
I believe this has been renamed to `actionclass`

Reply:
Will update this